libs/__tests__/deploy.test.js

```diff
@@ -0,0 +1,18 @@
+const utils = require('../utils');
+const deploy = require('../deploy').deploy;
+
+jest.mock('../utils');
+jest.useFakeTimers();
```

Comment:
Move this to the jest config. We should always be using fake timers

Reply:
Don't have a jest config yet, would be good to add one though :wink: